data/json/items/armor.json

```diff
@@ -163,6 +163,24 @@
     "material_thickness": 1,
     "flags": [ "WATER_FRIENDLY", "OUTER", "SPLINT" ]
   },
+  {
+    "id": "arm_xlsplint",
+    "type": "ARMOR",
+    "name": "arm splint",
+    "description": "A tool to help set bones and hold them in place.",
+    "weight": 300,
```

Comment:
Weight, volume, etc should potentially all be increased due to the new size and ingredients that go into it. Although you might want to reference other XL gear as to the approach (whether it's "proportional" flag or other tactic)